src/components/Notification/subcomponents/CompactNotification/CompactNotification.style.ts

```diff
@@ -0,0 +1,66 @@
+import { Theme } from '../../../../theme';
+import { rem, transparentize } from 'polished';
+import { NotificationTypes } from '../../Notification';
+import { CompactNotificationVariants } from './CompactNotification';
+import { css, SerializedStyles } from '@emotion/core';
+
+const typeToThemePalette = (theme: Theme, type: NotificationTypes) =>
+  type === 'success'
+    ? theme.palette.success['400']
+    : type === 'error'
+    ? theme.palette.error['400']
+    : type === 'info'
+    ? theme.palette.info['400']
+    : theme.palette.warning['400'];
+
+export const notificationsContainer = (
+  withFilling: boolean,
+  variant: CompactNotificationVariants,
+  type: NotificationTypes
+) => (theme: Theme): SerializedStyles => css`
+  display: flex;
+  justify-content: space-between;
+  overflow: hidden;
+  width: ${variant === 'inline' ? '100%' : rem(489)};
```

Comment:
i dont think `rem(489)` is correct

Reply:
This is something leftover while investigating the banner component.